### ISSUE TITLE
Introduce a player widget

### ISSUE THREAD

**Issue by kaikreuzer** [eclipse/smarthome#635]
migrated from Bugzilla [#464147](https://bugs.eclipse.org/bugs/show_bug.cgi?id=464147)
status NEW severity *enhancement* in component *UI* for *---*
Reported in version *unspecified* on platform *All*
Assigned to: Project Inbox

On 2015-04-08 07:21:54 -0400, Kai Kreuzer wrote:

> We have introduced a Player item type, which is supported through sitemaps with a default player widget so far (that is a switch widget with predefined button mappings).
> 
> To allow nicer and dedicated UI controls for players, we should also introduce a Player widget. See https://www.eclipse.org/forums/index.php/t/1065298/

**Comment by lolodomo** [comment]
@kaikreuzer : to be closed ?

**Comment by htreu** [comment]
another ping @kaikreuzer: is this still valid?

**Comment by kaikreuzer** [comment]
Sure, why not?

**Comment by htreu** [comment]
from BasicUI:

<img width="549" alt="screen shot 2017-08-29 at 19 22 07" src="https://user-images.githubusercontent.com/1729268/29834534-6aa31a4a-8cef-11e7-8439-149ae0e5d525.png">

do we miss one in PaperUI? Can you give some details please?

**Comment by htreu** [comment]
I guess we do miss a `Player` keyword in the DSL. The screenshot is with `Default player` in the sitemap file.

**Comment by kaikreuzer** [comment]
Yes, it is mainly about adding a Player option in the Sitemap DSL (plus its parameters, whatever makes sense for it) and adding support for it in all UIs (Classic+Basic) (best including the native openHAB UIs as well). 

@digitaldan As you were the one asking for such a widget at the time, does it still make sense to you and would it be easy to include support for such a new widget type in iOS and Android app?
